### PR TITLE
Update SDK to support Composite Slice implementation.

### PR DESCRIPTION
**Pull request by benembery**
Hope this is something like what you are looking for. 
I've tried to copy the implementation from other languages.

In response to #13

### CODE REVIEW

**Comment by srenault** [comment]
Thanks for your contribution. The `Fragment.cs` seems to have a big because tabs were replaced by spaces. Could you recover that change?

**Comment by benembery** [comment]
Sorry about this what size tab stops do you want, 4 or 2?

**Comment by srenault** [comment]
I believe we use `4`.

File: src/prismic/Fragment.cs [hunk 1]
```diff
@@ -484,7 +484,15 @@ public class Embed: Fragment {
...
-		public class Slice
+		public interface ISlice : Fragment
+		{
+			string SliceType { get; }
+			string SliceLabel { get; }
+
+			string AsHtml(DocumentLinkResolver resolver);
+		}
+
+		public class Slice : ISlice
```
Comment (srenault) [inline src/prismic/Fragment.cs]: As in others kits implementations, could you rename `Slice` to `SimpleSlice` ?

Comment (benembery) [inline src/prismic/Fragment.cs]: Sure. Would you like the `ISlice` interface to be renamed to `Slice` in keeping with the current coding style?

**Pull request merged**